sq/src/commands/keyring.rs

```diff
@@ -216,6 +234,111 @@ fn filter<F>(inputs: Option<clap::Values>, output: &mut dyn io::Write,
     Ok(())
 }
 
+/// Import keyring into DSM.
+fn dsm_import(config: Config, m: &ArgMatches) -> Result<()> {
```

Comment:
Q: Do you think it would be possible to deduplicate code? This function is using similar code as the (non-keyring) `dsm_import`